assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -141,6 +142,18 @@ export default function SettingsView() {
 					</p>
 				</div>
 			</div>
+
+			<div className="googlesitekit-settings-module__meta-items">
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						{ __( 'Ads Conversion ID', 'google-site-kit' ) }
+					</h5>
+					<p className="googlesitekit-settings-module__meta-item-data">
+						<DisplaySetting value={ adsConversionID } />
+					</p>
+				</div>
+			</div>
```

Comment:
Probably better to only show this if `useSnippet`, the same as in `SettingsEdit` and like we do here for e.g. Anonymize IP setting.

Reply:
Should we not also consider `canUseSnippet`? Wondering since `useSnippet` will remain `true` if GTM takes over adding the tag for Analytics even though Analytics isn't actually outputting the tag.

Reply:
Makes sense to check for `canUseSnippet` too. There are a few other places where we should probably add that (separate issue though).